client/src/main/java/com/alibaba/nacos/client/config/http/ServerHttpAgent.java

```diff
@@ -465,12 +480,12 @@ public String getCode() {
         @Override
         public String toString() {
             return "STSCredential{" +
-                "accessKeyId='" + accessKeyId + '\'' +
-                ", accessKeySecret='" + accessKeySecret + '\'' +
-                ", expiration=" + expiration +
-                ", securityToken='" + securityToken + '\'' +
-                ", lastUpdated=" + lastUpdated +
-                ", code='" + code + '\'' +
+                "accessKeyId='" + this.accessKeyId + '\'' +
+                ", accessKeySecret='" + this.accessKeySecret + '\'' +
+                ", expiration=" + this.expiration +
+                ", securityToken='" + this.securityToken + '\'' +
+                ", lastUpdated=" + this.lastUpdated +
+                ", code='" + this.code + '\'' +
```

Comment:
When using member variables normally, do not need to add `this`, otherwise the code looks very redundant. 
`this` can be used uniformly in the set method and constructor.

Reply:
"this" keywords aims to keep strong unified coding style.

Reply:
I know that, I mean it's enough for keep uniformly in constructor and set method.